extensions/ql-vscode/src/stories/model-editor/LibraryRow.stories.tsx

```diff
@@ -22,6 +22,7 @@ LibraryRow.args = {
       library: "sql2o",
       libraryVersion: "1.6.0",
       signature: "org.sql2o.Connection#createQuery(String)",
+      endpointType: EndpointType.Method,
```

Comment:
I don't know what the `EndpointType.Class` should "look" like. Is it worth having an example of that one somewhere? (e.g. in stories or in tests)

Reply:
I think that will come naturally as the endpoint types start getting used, so I don't think it's necessary for now.